dipy/segment/metric.py

```diff
@@ -1,15 +1,23 @@
 from dipy.segment.featurespeed import (Feature,
                                        IdentityFeature,
-                                       CenterOfMassFeature)
+                                       ResampleFeature,
+                                       CenterOfMassFeature,
+                                       MidpointFeature,
+                                       ArcLengthFeature,
+                                       VectorBetweenEndpointsFeature)
 
 from dipy.segment.metricspeed import (Metric,
                                       SumPointwiseEuclideanMetric,
                                       AveragePointwiseEuclideanMetric,
-                                      MinimumAverageDirectFlipMetric)
+                                      MinimumAverageDirectFlipMetric,
+                                      CosineMetric)
 
 from dipy.segment.metricspeed import (dist,
                                       distance_matrix)
 
+# Creates aliases
+EuclideanMetric = SumPointwiseEuclideanMetric
```

Comment:
Would this be confusing if someone were to look for the docstring of `EuclideanMetric` and found the docstring of `SumPointWiseEuclideanMetric`? Maybe better to implement explicitly as a class with its own docstring, but inheriting all of its properties from the parent? 

I am not sure - just thinking out loud here.